controllers/sample_controller.py

```diff
@@ -4,7 +4,7 @@
 from config import Config
 
 
-def load_sample(test_file):
-    browser.visit('{}/surveys/{}/{}'.format(Config.RESPONSE_OPERATIONS_UI, 'Bricks', '201801'))
+def load_sample(survey_name, period, test_file):
+    browser.visit(f'{Config.RESPONSE_OPERATIONS_UI}/surveys/{survey_name}/{period}')
```

Comment:
These were being kept consistent across the repo, I think it's worth keeping it consistent.